app/scripts/proactive/templates/file-browser-template.html

```diff
@@ -34,7 +34,7 @@ <h5>Browse, Upload files in <%=locationDescription%>, Select one file as the var
                 </ol>
             </div>
             <br>
-            <div id="files-container" style="overflow-x: hidden; overflow-y: scroll; height: 270px; overflow-wrap: break-word;">
+            <div id="files-container" style="overflow-x: hidden; overflow-y: scroll; height: 30em; overflow-wrap: break-word;">
```

Comment:
I think it would be best to use vh instead of em, it can this way adapt to different screens. @medou-boushab do you confirm ?

Reply:
@tao-xinxiu  I think in studio we can use max-height instead of height, this will reduce the empty space